app/models/in_person_enrollment.rb

```diff
@@ -126,6 +126,15 @@ def days_to_due_date
     (today...due_date).count
   end
 
+  def set_notification_sent_at
+    self.notification_sent_at = Time.zone.now
+    notification_phone_configuration.destroy if notification_phone_configuration.present?
+  end
```

Comment:
It looks like the `notification_sent_at` may not be persisted in this version of the change. Can you verify whether an update query is run for that field?

Reply:
Not valid anymore after refactor